cfgov/legacy/templates/complaint/complaint-landing.html

```diff
@@ -35,13 +35,147 @@
+                Learn more
+              </a>
+            </p>
+            <p>
+              We don’t verify all the allegations in complaint narratives.
+              <a href="#what-you-should-consider">
+                Learn more
+              </a>
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="block">
+        <div class="section-one ccdb-landing_section content-l">
+          <div class="content-l_col content-l_col-1-3">
+            <img class="isocon" alt="Complaint data" src="{% static 'complaint/img/view.png' %}">
+            <p>View, sort, and filter data right in your browser.</p>
+            <div class="btn-holder">
+              <p><a href="/data-research/consumer-complaints/search/" class="btn">View complaint data</a></p>
+            </div>
+          </div>
+          <div class="content-l_col content-l_col-1-3">
+            <img class="isocon" alt="Download data and access the API" src="{% static 'complaint/img/download.png' %}">
+            <p>All complaint data we publish is freely available for anyone to use, analyze, and build on.</p>
+            <div class="btn-holder">
+              <p><a href="#download-the-data" class="btn">Download options and API</a></p>
+            </div>
+          </div>
+          <div class="content-l_col content-l_col-1-3">
+            <img class="isocon" alt="Consumer narratives" src="{% static 'complaint/img/analyze.png' %}">
+            <p>We publish reports on complaints and share information with state and federal agencies.</p>
+            <div class="btn-holder">
+              <p><a href="/data-research/research-reports/?title=&categories=consumer-complaint&from_date=&to_date=" class="btn">Read complaint reports</a></p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="block block__border-top block__border-bottom block__padded-top block__padded-bottom block__flush-bottom">
+        <div class="section-two ccdb-landing_section content-l">
+          <div class="content-l_col content-l_col-2-3">
+            <h2 id="about-the-database">About the database</h2>
+            <div class="db-numbers callouts-small">
+              <p class="callout"><span class="numeric-callout percent-timely">97%</span> of complaints sent to companies get timely responses</p>
+            </div>
+            <p>The Consumer Complaint Database is a collection of complaints about consumer financial products and services that we sent to companies for response. Complaints are published after the company responds, confirming a commercial relationship with the consumer, or after 15 days, whichever comes first. Complaints referred to other regulators, such as complaints about depository institutions with less than $10 billion in assets, are not published in the Consumer Complaint Database. The database generally updates daily.</p>
+            <p>
+              <span class="publication-criteria">
```

Comment:
Can this span be removed and `class="publication-criteria"` added to the parent paragraph?

Reply:
Yep, looks like `class="publication-criteria"` was outdated and no longer used. I removed the `<span>` entirely.